addons/payment/views/account_journal_views.xml

```diff
@@ -7,8 +7,9 @@
         <field name="inherit_id" ref="account.view_account_journal_form"/>
         <field name="arch" type="xml">
             <xpath expr="//field[@name='inbound_payment_method_line_ids']//field[@name='payment_account_id']" position="after">
-                <field name="payment_acquirer_id" invisible="1"/>
+                <field name="payment_acquirer_id" options="{'no_open': True, 'no_create': True}" optional="hide"/>
```

Comment:
I imagine we're not the only ones to get hit by failing Odoo.sh deploys on
```
Field 'code' used in domain of field 'payment_provider_id' ([('code', '=', code)]) must be present in view but is missing.
```
when updating a view that inherits from `account.view_account_journal_form`.